proto/ethermint/evm/v1/query.proto

```diff
@@ -221,6 +221,8 @@ message EthCallRequest {
   bytes args = 1;
   // the default gas cap to be used
   uint64 gas_cap = 2;
+  // the proposer of the requested block
+  bytes proposer_address = 3;
```

Comment:
let's use the `(gogoproto.customtype)` option so that this translates to `sdk.ConsAddress` directly